dbms/src/Storages/DeltaMerge/ReadThread/SegmentReader.h

```diff
@@ -66,6 +66,10 @@ class SegmentReaderPoolManager
     void addTask(MergedTaskPtr && task);
     bool isSegmentReader() const;
 
+    // Explicitly, release reader_pools and reader_ids.
+    // The destructor will do this automatically.
+    // This function is for unittests.
+    void stop();
```

Comment:
should we also call this method in Server.cpp?
https://github.com/pingcap/tiflash/blob/master/dbms/src/Server/Server.cpp#L1219-L1228